Improve SQAaaS badge

At the moment the SQAaaS badge is silver. Unit testing and code quality should be passing. Need to investigate.

Unite Testing fails because there are some `python3` and `pip3` commands in `env-files/torch/generic_torch.sh`
Ref: https://jenkins.eosc-synergy.eu/job/eosc-synergy-org/job/itwinai.assess.sqaaas/job/main/382/execution/node/133/log/?consoleFull

@matbun while testing the new image for the SQAaaS action, I found out that also `python3-venv` and `cmake` are required as well. Currently waiting for the [SQAaaS action run](https://github.com/interTwin-eu/itwinai/actions/runs/11011643417).

Regarding the code style check, I've executed `flake8` with with the config file at `.github/linters/.flake8` (hope this is the right one to be symlnked) and linting issues still appear. Could you solve/add exceptions to these errors? Thanks!

The full log:
```bash
./use-cases/xtclim/train.py:6:1: F401 'typing.Dict' imported but unused
./use-cases/xtclim/train.py:7:1: F401 'argparse' imported but unused
./use-cases/xtclim/train.py:8:1: F401 'logging' imported but unused
./use-cases/xtclim/train.py:9:1: F401 'datetime.datetime' imported but unused
./use-cases/xtclim/train.py:14:1: E402 module level import not at top of file
./use-cases/xtclim/train.py:30:1: W391 blank line at end of file
./use-cases/xtclim/src/model.py:5:16: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:6:18: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:7:19: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:8:17: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:25:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/src/model.py:29:89: E501 line too long (93 > 88 characters)
./use-cases/xtclim/src/model.py:33:89: E501 line too long (95 > 88 characters)
./use-cases/xtclim/src/model.py:53:89: E501 line too long (95 > 88 characters)
./use-cases/xtclim/src/model.py:57:89: E501 line too long (95 > 88 characters)
./use-cases/xtclim/src/model.py:61:89: E501 line too long (94 > 88 characters)
./use-cases/xtclim/src/model.py:71:37: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:72:36: E261 at least two spaces before inline comment
./use-cases/xtclim/src/model.py:73:34: E261 at least two spaces before inline comment
./use-cases/xtclim/src/utils.py:9:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/utils.py:14:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/utils.py:14:58: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/utils.py:14:60: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/utils.py:18:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/utils.py:18:36: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/utils.py:18:38: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/utils.py:22:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/utils.py:22:50: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/utils.py:22:52: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/trainer.py:10:89: E501 line too long (94 > 88 characters)
./use-cases/xtclim/src/trainer.py:11:89: E501 line too long (96 > 88 characters)
./use-cases/xtclim/src/trainer.py:27:1: F401 'utils.save_reconstructed_images' imported but unused
./use-cases/xtclim/src/trainer.py:27:1: F401 'utils.save_ex' imported but unused
./use-cases/xtclim/src/trainer.py:30:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/trainer.py:52:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:53:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:54:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:55:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:56:9: F841 local variable 'stop_delta' is assigned to but never used
./use-cases/xtclim/src/trainer.py:57:9: F841 local variable 'patience' is assigned to but never used
./use-cases/xtclim/src/trainer.py:58:9: F841 local variable 'early_count' is assigned to but never used
./use-cases/xtclim/src/trainer.py:59:9: F841 local variable 'old_valid_loss' is assigned to but never used
./use-cases/xtclim/src/trainer.py:74:89: E501 line too long (94 > 88 characters)
./use-cases/xtclim/src/trainer.py:82:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/src/trainer.py:85:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/src/trainer.py:102:89: E501 line too long (89 > 88 characters)
./use-cases/xtclim/src/trainer.py:115:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:122:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:127:13: F841 local variable 'lr' is assigned to but never used
./use-cases/xtclim/src/trainer.py:127:18: F821 undefined name 'lr'
./use-cases/xtclim/src/trainer.py:129:1: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:132:9: E303 too many blank lines (2)
./use-cases/xtclim/src/trainer.py:140:9: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:141:17: E116 unexpected indentation (comment)
./use-cases/xtclim/src/trainer.py:143:17: E116 unexpected indentation (comment)
./use-cases/xtclim/src/trainer.py:143:17: E265 block comment should start with '# '
./use-cases/xtclim/src/trainer.py:150:1: E265 block comment should start with '# '
./use-cases/xtclim/src/anomaly.py:9:1: F401 'initialization.beta' imported but unused
./use-cases/xtclim/src/anomaly.py:21:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:25:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:30:21: E201 whitespace after '['
./use-cases/xtclim/src/anomaly.py:30:23: E201 whitespace after '('
./use-cases/xtclim/src/anomaly.py:30:81: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:31:25: E127 continuation line over-indented for visual indent
./use-cases/xtclim/src/anomaly.py:31:43: E202 whitespace before ')'
./use-cases/xtclim/src/anomaly.py:31:69: E202 whitespace before ']'
./use-cases/xtclim/src/anomaly.py:40:20: E201 whitespace after '['
./use-cases/xtclim/src/anomaly.py:40:22: E201 whitespace after '('
./use-cases/xtclim/src/anomaly.py:40:79: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:41:25: E127 continuation line over-indented for visual indent
./use-cases/xtclim/src/anomaly.py:41:42: E202 whitespace before ')'
./use-cases/xtclim/src/anomaly.py:41:67: E202 whitespace before ']'
./use-cases/xtclim/src/anomaly.py:45:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:46:40: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:48:85: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:49:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:49:74: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:50:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:50:67: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:51:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:52:82: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:53:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:53:80: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:54:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:56:83: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:57:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:57:81: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:58:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:62:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:62:80: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:63:53: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:70:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:71:89: E501 line too long (95 > 88 characters)
./use-cases/xtclim/src/anomaly.py:72:89: E501 line too long (93 > 88 characters)
./use-cases/xtclim/src/anomaly.py:79:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:85:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:88:89: E501 line too long (99 > 88 characters)
./use-cases/xtclim/src/anomaly.py:90:24: E201 whitespace after '['
./use-cases/xtclim/src/anomaly.py:90:26: E201 whitespace after '('
./use-cases/xtclim/src/anomaly.py:90:83: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:91:29: E127 continuation line over-indented for visual indent
./use-cases/xtclim/src/anomaly.py:91:46: E202 whitespace before ')'
./use-cases/xtclim/src/anomaly.py:91:71: E202 whitespace before ']'
./use-cases/xtclim/src/anomaly.py:98:86: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:99:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:99:84: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:100:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:101:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:103:84: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:104:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:104:84: W291 trailing whitespace
./use-cases/xtclim/src/anomaly.py:105:57: E128 continuation line under-indented for visual indent
./use-cases/xtclim/src/anomaly.py:108:1: W293 blank line contains whitespace
./use-cases/xtclim/src/anomaly.py:113:89: E501 line too long (107 > 88 characters)
./use-cases/xtclim/src/anomaly.py:114:89: E501 line too long (97 > 88 characters)
./use-cases/xtclim/src/anomaly.py:114:98: W292 no newline at end of file
./use-cases/xtclim/src/engine.py:5:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/engine.py:20:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/engine.py:37:89: E501 line too long (94 > 88 characters)
./use-cases/xtclim/src/engine.py:46:24: E261 at least two spaces before inline comment
./use-cases/xtclim/src/engine.py:49:40: E261 at least two spaces before inline comment
./use-cases/xtclim/src/engine.py:52:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/engine.py:69:89: E501 line too long (98 > 88 characters)
./use-cases/xtclim/src/engine.py:71:17: E225 missing whitespace around operator
./use-cases/xtclim/src/engine.py:83:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/engine.py:84:45: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/engine.py:84:47: E251 unexpected spaces around keyword / parameter equals
./use-cases/xtclim/src/engine.py:105:89: E501 line too long (98 > 88 characters)
./use-cases/xtclim/src/engine.py:107:17: E225 missing whitespace around operator
./use-cases/xtclim/src/engine.py:115:39: E261 at least two spaces before inline comment
./use-cases/xtclim/src/engine.py:121:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/src/engine.py:138:89: E501 line too long (98 > 88 characters)
./use-cases/xtclim/src/engine.py:140:17: E225 missing whitespace around operator
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:18:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:26:5: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:70:89: E501 line too long (91 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:74:89: E501 line too long (91 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:78:89: E501 line too long (91 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:82:89: E501 line too long (91 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:105:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:111:9: E265 block comment should start with '# '
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:121:9: E303 too many blank lines (2)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:121:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:131:9: E303 too many blank lines (2)
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:131:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_2d_seasons.py:134:89: E501 line too long (90 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:21:1: E302 expected 2 blank lines, found 1
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:31:89: E501 line too long (97 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:92:89: E501 line too long (91 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:115:5: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:149:89: E501 line too long (90 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:157:5: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:181:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:188:89: E501 line too long (89 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:191:89: E501 line too long (89 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:194:89: E501 line too long (89 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:205:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:208:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:211:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:214:89: E501 line too long (92 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:224:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:229:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:236:9: E303 too many blank lines (2)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:262:9: E303 too many blank lines (2)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:262:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:275:9: E303 too many blank lines (2)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:275:9: E266 too many leading '#' for block comment
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:280:9: E265 block comment should start with '# '
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:281:9: E265 block comment should start with '# '
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:282:9: E265 block comment should start with '# '
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:289:89: E501 line too long (96 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:292:89: E501 line too long (96 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:295:89: E501 line too long (96 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_functions_2d_ssp.py:298:89: E501 line too long (96 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_3d_seasons.py:23:1: E265 block comment should start with '# '
./use-cases/xtclim/preprocessing/preprocess_3d_seasons.py:80:89: E501 line too long (90 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_3d_seasons.py:84:89: E501 line too long (90 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_3d_seasons.py:88:89: E501 line too long (90 > 88 characters)
./use-cases/xtclim/preprocessing/preprocess_3d_seasons.py:92:89: E501 line too long (90 > 88 characters)
./use-cases/3dgan/__init__.py:1:1: E266 too many leading '#' for block comment
./use-cases/3dgan/__init__.py:1:46: W292 no newline at end of file
./use-cases/3dgan/__init__.py:1:46: W292 no newline at end of file
./use-cases/3dgan/__init__.py:1:46: W292 no newline at end of file
./use-cases/cyclones/trainer.py:9:1: F401 'itwinai.tensorflow.distributed.get_strategy' imported but unused
./use-cases/cyclones/cyclones_vgg.py:591:1: E128 continuation line under-indented for visual indent
./use-cases/cyclones/cyclones_vgg.py:594:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:603:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:612:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:621:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:630:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:639:9: F821 undefined name 'ConvBlock'
./use-cases/cyclones/cyclones_vgg.py:669:1: E122 continuation line missing indentation or outdented
```

--- a/.github/workflows/sqaaas.yml
+++ b/.github/workflows/sqaaas.yml
@@ -24,7 +24,7 @@ jobs:
             make torch-env-cpu
             make tensorflow-env-cpu
             .venv-pytorch/bin/pytest -v ./tests/ -m "not slurm and not memory_heavy"
-          container: eoscsynergy/sqaaas-micromamba:1.5.3-1
+          container: eoscsynergy/sqaaas-micromamba:1.5.3-1-rc.8
       - name: Print out payload
         run: cat workflow_validation_step.json
       - name: SQAaaS assessment with unit testing (QC.Uni) step

